src/index.js

```diff
@@ -1,14 +1,70 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+let newTask;
+const arrowBtn = document.getElementById('arrow');
+const refresh = document.getElementById('refresh');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+class Todolist {
+  constructor(description, completed, index) {
```

Comment:
Please make sure that a function for editing task descriptions is implemented as requested by the previous reviewer.

- I should be able to modify a particular task and update it. You can achieve this by making the task description element an `input` element. Then you can apply an on `change` event on the input field to ensure that the new value is saved.

For example:
```
<input type="text" name="description" id="description" value={// here your set the desriptiion/task as the value} />

const task = document.getElementById('description');

task.addEventListener('change', (e) => {
// get onChange value
const newTask = e.target.value

// Update the local storage array with the new value
});
```